components/tracker/src/lib.rs

```diff
@@ -118,6 +121,9 @@ pub enum RequestType {
 #[derive(Debug, Default, Clone)]
 pub struct RequestMetrics {
     pub get_snapshot_nanos: u64,
+    pub read_index_propose_wait_nanos: u64,
+    pub read_index_confirm_wait_nanos: u64,
+    pub read_pool_schedule_wait_nanos: u64,
```

Comment:
Should we record this for coprocessor requests too using the `schedule_wait_time` in the [coprocessor tracker](https://github.com/tikv/tikv/blob/master/src/coprocessor/tracker.rs#L53)?

Reply:
Added~